python/paddle/fluid/layers/nn.py

```diff
@@ -4481,6 +4483,89 @@ def reshape(x, shape, actual_shape=None, act=None, inplace=True, name=None):
     return helper.append_activation(out)
 
 
+def squeeze(input, axes, inplace=False, name=None):
```

Comment:
the parameter `inplace` seem to be useless, and maybe it should be removed from `squeeze_op.cc` and `unsqueeze_op.cc`.  
 This parameter was needed before because these two ops invoked `reshape_op`, and `reshape_op` has this parameter, but now this parameter has been removed in `reshape_op`.